Domain/sdLitica.Entities/Analytics/UserAnalyticsOperation.cs

```diff
@@ -25,23 +27,23 @@ public UserAnalyticsOperation()
         }
 
 
-
-        public Guid Id { get; set; } // temporary
-
         /// <summary>
         /// Name of operation to be performed. 
         /// </summary>
+        [Column("OPERATION_NAME")]
         public string OpName { get; set; }
```

Comment:
Longer names are always better to read, and they wont impact code size and performance.

OpName -> OperationName

Reply:
forgot about that one. will rename